### PR TITLE
Mark that TopologyAwareHints feature is default on in 1.24

### PR DESCRIPTION
**Pull request by tengqm**
xref: kubernetes/kubernetes#108747

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Mark that TopologyAwareHints feature is default on in 1.24

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -206,7 +206,8 @@ different Kubernetes components.
 | `SuspendJob` | `false` | Alpha | 1.21 | 1.21 |
 | `SuspendJob` | `true` | Beta | 1.22 | |
 | `TopologyAwareHints` | `false` | Alpha | 1.21 | 1.22 |
-| `TopologyAwareHints` | `false` | Beta | 1.23 | |
+| `TopologyAwareHints` | `false` | Beta | 1.23 | 1.23 |
+| `TopologyAwareHints` | `true` | Beta | 1.23 | |
```
Comment (jihoon-seo) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:210]: 1.23 vs. 1.24, which one is right? 😊

```suggestion
| `TopologyAwareHints` | `false` | Beta | 1.23 | 1.23 |
| `TopologyAwareHints` | `true` | Beta | 1.24 | |
```

Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: my bad.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -206,7 +206,8 @@ different Kubernetes components.
 | `SuspendJob` | `false` | Alpha | 1.21 | 1.21 |
 | `SuspendJob` | `true` | Beta | 1.22 | |
 | `TopologyAwareHints` | `false` | Alpha | 1.21 | 1.22 |
-| `TopologyAwareHints` | `false` | Beta | 1.23 | |
+| `TopologyAwareHints` | `false` | Beta | 1.23 | 1.23 |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:209]: https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/ doesn't mention a step-to-enable. Was this feature previously off by default?

If so, then _ideally_ we also update https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/ before the v1.24 release, to show that you need to enable the feature gate to have it work.

Comment (reylejano) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:209]: https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/  has been updated by @tengqm  in PR https://github.com/kubernetes/website/pull/32413/

**Comment by mehabhalodiya** [comment]
enhancement ref: https://github.com/kubernetes/enhancements/issues/2433

**Pull request merged**